projects/akita-ng-fire/src/lib/collection-group/collection-group.service.ts

```diff
@@ -50,13 +59,14 @@ export abstract class CollectionGroupService<S extends EntityState> {
       setLoading(storeName, true);
     }
     return this.db.collectionGroup(this.collectionId, query).stateChanges().pipe(
-      withTransaction(actions => syncStoreFromDocAction(storeName, actions, this.idKey))
+      map(value => this.formatFromFirestore(value)),
+      withTransaction(actions => syncStoreFromDocAction(storeName, actions, this.idKey, this.formatFromFirestore))
```

Comment:
Fourth parameter should be a arrow function to avoid bind issue : 
```typescript
withTransaction(actions => syncStoreFromDocAction(storeName, actions, this.idKey, (entity) => this.formatFromFirestore(entity)))
```